ocs_ci/helpers/network_policy_helpers.py

```
@@ -0,0 +1,386 @@
+    pod_obj = get_pod_obj(source_pod_name, namespace=namespace)
+    cmd = (
+        f"bash -c 'timeout {timeout} "
+        f"bash -c \"echo > /dev/tcp/{target_ip}/{port}\" "
+        f"&& echo CONNECTED || echo FAILED'"
+    )
+    try:
+        result = pod_obj.exec_cmd_on_pod(
+            cmd, out_yaml_format=False, timeout=timeout + 10
+        )
+        connected = "CONNECTED" in str(result)
+    except (CommandFailed, Exception):
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)